File: 351003/Gorodko/app/Dto/ReactionResponseTo.cs
using System.Text.Json.Serialization;

namespace Project.Dto {
    public class ReactionResponseTo : BaseResponseTo {
        [JsonPropertyName("tweetId")]
        public long TweetId { get; set; }

        [JsonPropertyName("content")]
        public string Content { get; set; } = string.Empty;
    }
}

File: 351002/Gaiduk/publisher/src/main/java/com/example/publisher/dto/ReactionResponseTo.java
package com.example.publisher.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ReactionResponseTo {
    private Long id;
    private Long tweetId;
    private String content;
}